PowerEditor/src/WinControls/AboutDlg/URLCtrl.h

```diff
@@ -26,17 +26,19 @@ class URLCtrl : public Window {
 	void create(HWND itemHandle, int cmd, HWND msgDest = NULL);
     void destroy();
 private:
+	HCURSOR& getCursor();
 	void action();
 protected :
     generic_string _URL;
     HFONT _hfUnderlined = nullptr;
     HCURSOR _hCursor = nullptr;
+	HCURSOR _hCursorPredefined = nullptr;

```

Comment:
Why don't we use system's one (ie. `::LoadCursor(nullptr, IDC_HAND);`) and remove XORMask & ANDMask ?

Reply:
Ok